tox.ini

```diff
@@ -451,11 +452,13 @@ deps =
     sanic-v20: sanic>=20.0,<21.0
     sanic-v21: sanic>=21.0,<22.0
     sanic-v22: sanic>=22.0,<22.9.0
+    sanic-v23: sanic==23.6
```

Comment:
If possible would be good to test with an interval here, too, so that we can catch potential issues with newer 23.* versions.

Also, this essentially reads as "we're not supporting sanic between 23.0 and 23.5", is that the case?

Reply:
I have updated the tox.ini to reflect that Sanic does not use semver